src/components/Form/UploadDocumentForm.jsx

```diff
@@ -84,47 +96,99 @@ const UploadDocumentForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleDocumentUpload} autoComplete="off">
-        <label htmlFor="verify-checkbox">
-          Verify File on upload:
-          <input
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleDocumentUpload} autoComplete="off">
+          <FormControlLabel
+            control={<Checkbox />}
+            label="Verify file on upload"
             id="verify-checkbox"
-            type="checkbox"
             value={state.verifyFile}
-            onClick={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
+            onChange={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
           />
-        </label>
-        <div style={formRowStyle}>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="upload-doc-expiration"
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expiration Date"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+                slotProps={{
+                  textField: {
+                    helperText: 'MM/DD/YYYY'
+                  }
+                }}
+              />
+            </LocalizationProvider>
+          </FormControl>
           <br />
-          <textarea id="upload-doc-desc" name="description" {...description} />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doctype">File to upload: </label>
-          <input
-            id="upload-doctype"
-            type="file"
-            name="uploadDoctype"
-            accept=".pdf, .docx, .doc, .txt, .rtf, .gif"
-            onChange={handleFileChange}
-          />
-          <button disabled={state.processing} type="submit">
-            Upload file
-          </button>
-        </div>
-      </form>
+          <br />
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* <Typography htmlFor="upload-doc-desc">Enter description:</Typography> */}
```

Comment:
Can remove these Lines for 137 to 138. The `<InputLabel>` is being handled by the label and placeholder attributes in `<TextField>`.

That means we can also remove the `id` attribute in Line 140.